web/docs/tutorials/todo-app/03-listing-tasks.md

```diff
@@ -39,6 +39,7 @@ query getTasks {
 }
 ```
 :::caution
+<!-- This block is mostly duplicated in typescript.md -->
```

Comment:
Maybe you can use this to keep it tidy: https://docusaurus.io/docs/markdown-features/react#importing-markdown

Reply:
Nice idea, but they're not really identical. I could make them identical, but then I think both pages would suffer content-wise.